### PR TITLE
Blog update for GSOC 2018 plus an assortment of minor cleanup

### PR DESCRIPTION
**Pull request by Cervator**
Could use a few more screenies/videos

### CODE REVIEW

**Comment by Cervator** [comment]
Let's preview this

Wonder if that still works

**Comment by nihal111** [comment]
Let's preview this

**Comment by gooey-splashsite** [comment]
Preview is available on http://preview.terasology.net/splashsite/pull/105 :rocket:

File: _posts/2018-09-02-gsoc-2018-results.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+The first big piece was better preparing the game world using an existing "spawn platform" (serving as a sort of lobby) and various artwork assets. Home bases exist for the two teams and you can teleport down to join a specific team. More world features are planned to make for a more interesting battlefield.
+
+A second large piece was making actual CTF functionality work, which is probably where most complexity snuck in, to make everything reliable in multiplayer. Players can steal flags from the enemy base, each other, and make a capture back at their home base. A player carrying a flag gets an awesome shower of particles to indicate they're the flag carrier! A double-screenie follows (flag carrier on left, flag chaser on right)
+
+![LightAndShadow](img/LnS1.jpg "LightAndShadow")
```
Comment (nihal111) [inline _posts/2018-09-02-gsoc-2018-results.md]: Please change this link to a relative link.
`{{ site.baseurl }}/img/LnS1.jpg` should work probably.

**Review by nihal111 — CHANGES_REQUESTED** [review]
Please make the image links relative.
It should work as is for all normal purposes, but the preview builds on a different baseurl- `http://preview.terasology.net/splashsite/pull/105/` in this case. So, absolute links fail.

**Comment by Cervator** [comment]
Let's preview this

**Comment by gooey-splashsite** [comment]
Preview is available on http://preview.terasology.net/splashsite/pull/105 :rocket:

**Comment by nihal111** [comment]
Let's preview this

**Comment by gooey-splashsite** [comment]
Preview is available on http://preview.terasology.net/splashsite/pull/105 :rocket:

**Comment by Cervator** [comment]
Let's preview this

**Comment by gooey-splashsite** [comment]
Preview is available on http://preview.terasology.net/splashsite/pull/105 :rocket:

**Pull request merged**